tests/e2e/alias_test.go

```diff
@@ -0,0 +1,242 @@
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/consensys/quorum-key-manager/pkg/client"
+	"github.com/consensys/quorum-key-manager/src/aliases/api/types"
+	"github.com/consensys/quorum-key-manager/tests"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type aliasTestSuite struct {
+	suite.Suite
+	err  error
+	ctx  context.Context
+	rand *rand.Rand
+
+	client *client.HTTPClient
+}
+
+func TestAlias(t *testing.T) {
+	cfg, err := tests.NewConfig()
+	require.NoError(t, err)
+
+	token, err := generateJWT("./certificates/client.key", "*:*", "e2e|keys_test")
+	require.NoError(t, err)
+
+	cl := client.NewHTTPClient(
+		&http.Client{
+			Transport: NewTestHttpTransport(token, "", nil),
+		}, &client.Config{
+			URL: cfg.KeyManagerURL,
+		})
+	s := aliasTestSuite{
+		client: cl,
+		ctx:    context.Background(),
+		rand:   rand.New(rand.NewSource(time.Now().UnixNano())),
+	}
+	suite.Run(t, &s)
+}
+
+type testAlias struct {
+	reg    types.RegistryName
+	key    types.AliasKey
+	val    types.AliasValue
+	newVal types.AliasValue
+}
+
+func (s *aliasTestSuite) fakeAlias() testAlias {
+	randInt := s.rand.Intn(1 << 32)
+	randID := strconv.Itoa(randInt)
+	return testAlias{
+		reg:    types.RegistryName("JPM-" + randID),
+		key:    types.AliasKey("GoldmanSachs-" + randID),
+		val:    `["ROAZBWtSacxXQrOe3FGAqJDyJjFePR5ce4TSIzmJ0Bc=","2T7xkjblN568N1QmPeElTjoeoNT4tkWYOJYxSMDO5i0="]`,
+		newVal: `["ZOAZBWtSacxXQrOe3FGAqJDyJjFePR5ce4TSIzmJ0Bc=","2T7xkjblN568N1QmPeElTjoeoNT4tkWYOJYxSMDO5i0="]`,
+	}
+}
+
+func (s *aliasTestSuite) TestFull() {
+	fakeAlias := s.fakeAlias()
+	s.Run("create", func() {
```

Comment:
Please use the e2e standards we use in other e2e test units https://github.com/ConsenSys/quorum-key-manager/blob/main/tests/e2e/ethereum_test.go#L112
```
- should {action} successfully
- should failed [with error type] {when|if} {failure scenario} 
```